<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginRegister</class>
 <widget class="QWidget" name="LoginRegister">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>712</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:24px;
}
QFrame
{
background:#333		;
border-radius:15px;
}
QPushButton
{
background:#F08080	;;
border-radius:15px;
}
QToolButton
{
background:#F08080	;;
border-radius:60px;
}
QLabel
{
color:white;
font-size:18px;
}

QPushButton:hover
{
color:#333;
border-radius:15px;
background:#ffffff	;
}
QLineEdit
{
background:transparent;
border:none;
color:#717072;
border-bottom:1px solid #717072;
}
QPushButton#back
{
border-radius:20px;
}</string>
  </property>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/usericon.png</normaloff>:/usericon.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>611</width>
     <height>471</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="AcceptUser">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>400</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsername">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>250</y>
      <width>461</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorUsername">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>190</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Error. Enter valid username</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelErrorPassword">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>310</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>century gothic</family>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Error. Enter valid password</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="showPassword">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/eye.png</normaloff>:/eye.png</iconset>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditMessage">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>360</y>
      <width>381</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/back.png</normaloff>:/back.png</iconset>
   </property>
  </widget>
  <zorder>back</zorder>
  <zorder>frame</zorder>
  <zorder>toolButton</zorder>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
